gophersnake.py

```diff
@@ -345,8 +346,8 @@ def load_fn():
 	elif entry[0] == "g":
 		# TO DO: open GIF files in own window.
 		def save_fn():
-			#load_with_status(entry, open_image_viewer)
-			save_with_status(entry[2], entry[3], int(entry[4]))
+			load_with_status(entry, open_image_viewer)
+			#save_with_status(entry[2], entry[3], int(entry[4]))
```

Comment:
Not sure what you will want to do here. I left your code as is and just swapped the comments.

Reply:
That was the right call to make, until Gophersnake becomes more sophisticated.